infrastructure/simulation-tool/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import config, { Config } from './config';
+
+const run = async () => {};
+
+export { run, config as simulatorConfig };
+
+export type { Config as SimulatorConfig };
```

Comment:
why do we export it?

Reply:
to get access to it in case we want to change the way the tool is used.

Reply:
I'd change it when needed